cassiopeia/solver/DistanceSolver.py

```diff
@@ -59,13 +59,15 @@ class DistanceSolver(CassiopeiaSolver.CassiopeiaSolver):
                 "inverse": Transforms each probability p by taking 1/p
                 "square_root_inverse": Transforms each probability by the
                     the square root of 1/p
+        threads: Number of threads to use for dissimilarity map computation.
```

Comment:
Maybe make this a more general threads argument? CCphylo supports multithreading with the -t argument so we could also set the number of threads used by the tree solver. Unless we anticipate cases where we want a different number of threads used by the dissimilarly computation and tree solver?

Reply:
Happy to change the docstring here.